vinyldns/resource_zone_test.go

```diff
@@ -22,26 +22,44 @@ import (
 	"github.com/vinyldns/go-vinyldns/vinyldns"
 )
 
+const (
+	zName             = "system-test."
+	zEmail            = "email@foo.com"
+	zEmailUpdated     = "updated_email@foo.com"
+	zConName          = "vinyldns."
+	zConKey           = "nzisn+4G2ldMn0q1CV3vsg=="
+	zConKeyName       = "vinyldns."
+	zConPrimaryServer = "vinyldns-bind9"
+	// NOTE: If ever the test config HCL is changed, these zACLRule*Hash var
```

Comment:
test config "ACL"?

Reply:
HCL is "Hashicorp Configuration Language" - The Terraform DSL itself.